unlock-app/src/components/interface/checkout/CardConfirmationCheckout.tsx

```diff
@@ -128,8 +129,13 @@ export const CardConfirmationCheckout = ({
   }
 
   const handleHasKey = (key: any) => {
-    setKeyExpiration(key.expiration)
```

Comment:
The `<Lock>` component will call this handler even when there is no key to make sure the state us propagated up.